### PR TITLE
Add Keychron Q11

### PR DESCRIPTION
**Pull request by adophoxia**
## Description
Add `ansi_encoder` (or in this case, `ANSI Knob`) variant of Keychron Q11.

## QMK Pull Request 

* https://github.com/qmk/qmk_firmware/pull/20125

## Checklist

- [x] The VIA support for this keyboard is **MERGED** in QMK master already **(MANDATORY)**
- [x] The VIA definition follows the guide here: https://caniusevia.com/docs/layouts
- [x] I have a V3 JSON version for this keyboard definition.**(MANDATORY)**
- [x] I have tested this keyboard definition using VIA's "Design" tab.
- [x] I have tested this keyboard definition with firmware on a device.
- [x] I have assigned alpha keys and modifier keys with the correct colors.
- [x] The Vendor ID is not `0xFEED`

### CODE REVIEW

File: v3/keychron/q11/q11_us_knob_v1.0.json [hunk 1]
```diff
@@ -0,0 +1,308 @@
...
+    {"name": "Siri",            "title": "Siri in macOS",            "shortName": "Siri"},
+    {"name": "Task View",       "title": "Task View in windows",     "shortName": "Task"},
+    {"name": "File Explorer",   "title": "File Explorer in windows", "shortName": "File"},
+    {"name": "Screen Shot",     "title": "Screenshot in macOS",      "shortName": "SShot"},
+    {"name": "Cortana",         "title": "Cortana in windows",       "shortName": "Cortana"}
+  ],
+  "matrix": {"rows": 12, "cols": 9},
+  "layouts": {
+    "keymap": [
+      [
```
Comment (lalalademaxiya1) [inline v3/keychron/q11/q11_us_knob_v1.0.json:83]:
```suggestion
      [
        {
          "c": "#aaaaaa"
        },
        "0,0\n\n\n\n\n\n\n\n\ne0",
        {
          "x": 0.25,
          "c": "#777777"
        },
        "0,1\nESC",
        {
          "c": "#cccccc"
        },
        "0,2",
        "0,3",
        "0,4",
        "0,5",
        {
          "c": "#aaaaaa"
        },
        "0,6",
        "0,7",
        {
          "x": 0.75
        },
        "6,0",
        "6,1",
        {
          "c": "#cccccc"
        },
        "6,2",
        "6,3",
        "6,4",
        "6,5",
        {
          "c": "#aaaaaa"
        },
        "6,6",
        "6,7",
        {
          "x": 0.25
        },
        "6,8\n\n\n\n\n\n\n\n\ne1"
      ],
      [
        {
          "y": 0.25
        },
        "1,0",
        {
          "x": 0.25
        },
        "1,1",
        {
          "c": "#cccccc"
        },
        "1,2",
        "1,3",
        "1,4",
        "1,5",
        "1,6",
        "1,7",
        {
          "x": 0.75
        },
        "7,0",
        "7,1",
        "7,2",
        "7,3",
        "7,4",
        "7,5",
        {
          "c": "#aaaaaa",
          "w": 2
        },
        "7,6",
        {
          "x": 0.25
        },
        "7,8"
      ],
      [
        "2,0",
        {
          "x": 0.25,
          "w": 1.5
        },
        "2,1",
        {
          "c": "#cccccc"
        },
        "2,2",
        "2,3",
        "2,4",
        "2,6",
        "2,7",
        {
          "x": 0.75
        },
        "8,0",
        "8,1",
        "8,2",
        "8,3",
        "8,4",
        "8,5",
        "8,6",
        {
          "c": "#aaaaaa",
          "w": 1.5
        },
        "8,7",
        {
          "x": 0.25
        },
        "8,8"
      ],
      [
        "3,0",
        {
          "x": 0.25,
          "w": 1.75
        },
        "3,1",
        {
          "c": "#cccccc"
        },
        "3,2",
        "3,3",
        "3,4",
        "3,5",
        "3,6",
        {
          "x": 0.75
        },
        "9,0",
        "9,1",
        "9,2",
        "9,3",
        "9,4",
        "9,5",
        {
          "c": "#777777",
          "w": 2.25
        },
        "9,7",
        {
          "x": 0.25,
          "c": "#aaaaaa"
        },
        "9,8"
      ],
      [
        "4,0",
        {
          "x": 0.25,
          "w": 2.25
        },
        "4,2",
        {
          "c": "#cccccc"
        },
        "4,3",
        "4,4",
        "4,5",
        "4,6",
        "4,7",
        {
          "x": 0.75
        },
        "10,0",
        "10,1",
        "10,2",
        "10,3",
        "10,4",
        {
          "c": "#aaaaaa",
          "w": 1.75
        },
        "10,5",
        {
          "x": 0.25,
          "c": "#777777"
        },
        "10,7"
      ],
      [
        {
          "c": "#aaaaaa"
        },
        "5,0",
        {
          "x": 0.25,
          "w": 1.25
        },
        "5,1",
        {
          "w": 1.25
        },
        "5,2",
        {
          "w": 1.25
        },
        "5,3",
        {
          "w": 1.25
        },
        "5,4",
        {
          "w": 2.25
        },
        "5,6",
        {
          "x": 0.75,
          "w": 2.75
        },
        "11,1",
        "11,2",
        "11,3",
        "11,4",
        {
          "x": 0.25,
          "c": "#777777"
        },
        "11,6",
        "11,7",
        "11,8"
      ]
```

Comment (lalalademaxiya1) [reply v3/keychron/q11/q11_us_knob_v1.0.json:83]: Hi, Here a gap is added between the left and right parts to make it look more like two parts.

Comment (lalalademaxiya1) [reply v3/keychron/q11/q11_us_knob_v1.0.json:83]: And the color of the key was adjusted to make it match the color of the standard keycap of the product.

Comment (adophoxia) [reply v3/keychron/q11/q11_us_knob_v1.0.json]: Gap added and colors matched, @lalalademaxiya1.

**Review by lalalademaxiya1 — APPROVED** [review]
Looks good.

**Comment by adophoxia** [comment]
Updated checklist now that the Q11 is merged into `qmk/master`.

**Review by Xelus22 — APPROVED** [review]

**Pull request merged**